cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpa.kt

```diff
@@ -127,6 +135,48 @@ private class DataStorageCcpaImpl(context: Context) : DataStorageCcpa {
             }
         }
 
+    override var gppData: Map<String, Any?>
+        get() {
+            val result = TreeMap<String, Any?>()
+            preference.all
+                .filter { it.key.startsWith(DataStorageGdpr.IABTCF_KEY_PREFIX) }
```

Comment:
Here you probably meant to use 
```
KEY_IABGPP_PREFIX
```

Instead of 
```
IABTCF_KEY_PREFIX
```

Reply:
@andresilveirah clearly noticed, cudos to you, I will also do another round of manual testing of this stuff